### PR TITLE
Correct name definition

### PR DESCRIPTION
**Pull request by ling1726**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 **CSS-in-JS with ahead-of-time compilation.**
 
+[_Griffel (German): stylus, pen, slate pencil; an antiquated writing tool, used in ancient times, prior to the appearance of blackboards._](https://en.wiktionary.org/wiki/Griffel)
+
 Griffel is CSS-in-JS with near-zero runtime, SSR support and styles defined with JavaScript objects.
```
Comment (layershifter) [inline README.md:7]:
```suggestion
[_Griffel (German): stylus, pen, slate pencil; an antiquated writing tool, used in ancient times, prior to the appearance of blackboards._](https://en.wiktionary.org/wiki/Griffel)

Griffel ([_German_]((https://en.wiktionary.org/wiki/Griffel)) for stylus/pen) is CSS-in-JS with near-zero runtime, SSR support and styles defined with JavaScript objects.
```

image

- Not sure that it's the most important thing in README 🙃
- The suggestion IMO looks more accurate

File: README.md [hunk 2]
```diff
@@ -2,7 +2,9 @@
 
 **CSS-in-JS with ahead-of-time compilation.**
 
-Griffel is CSS-in-JS with near-zero runtime, SSR support and styles defined with JavaScript objects.
+[_Griffel (German): stylus, pen, slate pencil; an antiquated writing tool, used in ancient times, prior to the appearance of blackboards._](https://en.wiktionary.org/wiki/Griffel)
```
Comment (ling1726) [inline README.md:5]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -2,7 +2,8 @@
 
 **CSS-in-JS with ahead-of-time compilation.**
 
-Griffel is CSS-in-JS with near-zero runtime, SSR support and styles defined with JavaScript objects.
+
+Griffel ([_German_]((https://en.wiktionary.org/wiki/Griffel)) for stylus/pen) is CSS-in-JS with near-zero runtime, SSR support and styles defined with JavaScript objects.
```
Comment (ling1726) [inline README.md:6]:
```suggestion
Griffel ([_German_](https://en.wiktionary.org/wiki/Griffel)) for stylus/pen)) is CSS-in-JS with near-zero runtime, SSR support and styles defined with JavaScript objects.
```

File: README.md [hunk 4]
```diff
@@ -2,7 +2,8 @@
 
 **CSS-in-JS with ahead-of-time compilation.**
 
-Griffel is CSS-in-JS with near-zero runtime, SSR support and styles defined with JavaScript objects.
+
+Griffel ([_German_](https://en.wiktionary.org/wiki/Griffel)) for stylus/pen)) is CSS-in-JS with near-zero runtime, SSR support and styles defined with JavaScript objects.
```
Comment (ling1726) [inline README.md:6]:
```suggestion
Griffel ([_German for styus/pen_](https://en.wiktionary.org/wiki/Griffel)) is CSS-in-JS with near-zero runtime, SSR support and styles defined with JavaScript objects.
```

File: README.md [hunk 5]
```diff
@@ -2,7 +2,8 @@
 
 **CSS-in-JS with ahead-of-time compilation.**
 
-Griffel is CSS-in-JS with near-zero runtime, SSR support and styles defined with JavaScript objects.
+
+Griffel ([_German for styus/pen_](https://en.wiktionary.org/wiki/Griffel)) is CSS-in-JS with near-zero runtime, SSR support and styles defined with JavaScript objects.
```
Comment (ling1726) [inline README.md:6]:
```suggestion
Griffel ([_German for stylus/pen_](https://en.wiktionary.org/wiki/Griffel)) is CSS-in-JS with near-zero runtime, SSR support and styles defined with JavaScript objects.
```

File: README.md [hunk 6]
```diff
@@ -2,7 +2,8 @@
 
 **CSS-in-JS with ahead-of-time compilation.**
 
-Griffel is CSS-in-JS with near-zero runtime, SSR support and styles defined with JavaScript objects.
+
```
Comment (layershifter) [inline README.md:5]:
```suggestion
```

**Review by layershifter — APPROVED** [review]

**Pull request merged**